spec/index.bs

```diff
@@ -1409,62 +1409,63 @@ interface GPUBufferUsage {
+                    - |descriptor|.{{GPUBufferDescriptor/usage}} contains no other [=usage flags=] except {{GPUBufferUsage/COPY_SRC}}.
+                - If |descriptor|.{{GPUBufferDescriptor/mappedAtCreation}} is true then
+                    |descriptor|.{{GPUBufferDescriptor/size}} must be a multiple of 4.
 
-  1. Let |b| be a new {{GPUBuffer}} object.
-  1. Set |b|.{{GPUBuffer/[[size]]}} to |descriptor|.{{GPUBufferDescriptor/size}}.
-  1. Set |b|.{{GPUBuffer/[[usage]]}} to |descriptor|.{{GPUBufferDescriptor/usage}}.
-  1. If |descriptor|.{{GPUBufferDescriptor/mappedAtCreation}} is true:
+                Issue(gpuweb/gpuweb#605): Explain that the resulting error buffer can still be mapped at creation.
 
-    1. Set |b|.{{GPUBuffer/[[mapping]]}} to a new {{ArrayBuffer}} of size |b|.{{GPUBuffer/[[size]]}}.
-    1. Set |b|.{{GPUBuffer/[[mapping_range]]}} to `[0, descriptor.size]`.
-    1. Set |b|.{{GPUBuffer/[[mapped_ranges]]}} to `[]`.
-    1. Set |b|.{{GPUBuffer/[[state]]}} to [=buffer state/mapped at creation=].
+                Issue(gpuweb/gpuweb#605): Explain what are a {{GPUDevice}}'s `[[allowed buffer usages]]`.
+            </div>
 
-  1. Else:
+        - Let |b| be a new {{GPUBuffer}} object.
+        - Set |b|.{{GPUBuffer/[[size]]}} to |descriptor|.{{GPUBufferDescriptor/size}}.
+        - Set |b|.{{GPUBuffer/[[usage]]}} to |descriptor|.{{GPUBufferDescriptor/usage}}.
+        - If |descriptor|.{{GPUBufferDescriptor/mappedAtCreation}} is true:
 
-    1. Set |b|.{{GPUBuffer/[[mapping]]}} to `null`.
-    1. Set |b|.{{GPUBuffer/[[mapping_range]]}} to `null`.
-    1. Set |b|.{{GPUBuffer/[[mapped_ranges]]}} to `null`.
-    1. Set |b|.{{GPUBuffer/[[state]]}} to [=buffer state/unmapped=].
+            - Set |b|.{{GPUBuffer/[[mapping]]}} to a new {{ArrayBuffer}} of size |b|.{{GPUBuffer/[[size]]}}.
+            - Set |b|.{{GPUBuffer/[[mapping_range]]}} to `[0, descriptor.size]`.
+            - Set |b|.{{GPUBuffer/[[mapped_ranges]]}} to `[]`.
+            - Set |b|.{{GPUBuffer/[[state]]}} to [=buffer state/mapped at creation=].
 
-  1. Set each byte of |b|'s allocation to zero.
-  1. Return |b|.
+            Else:
 
-  Note: it is valid to set {{GPUBufferDescriptor/mappedAtCreation}} to true without {{GPUBufferUsage/MAP_READ}}
-    or {{GPUBufferUsage/MAP_WRITE}} in {{GPUBufferDescriptor/usage}}. This can be used to set the buffer's
-    initial data.
+            - Set |b|.{{GPUBuffer/[[mapping]]}} to `null`.
+            - Set |b|.{{GPUBuffer/[[mapping_range]]}} to `null`.
+            - Set |b|.{{GPUBuffer/[[mapped_ranges]]}} to `null`.
+            - Set |b|.{{GPUBuffer/[[state]]}} to [=buffer state/unmapped=].
 
-  <div class=validusage>
-    <dfn abstract-op>createBuffer Valid Usage</dfn>
-      Given a {{GPUDevice}} |this| and a {{GPUBufferDescriptor}} |descriptor|
-      the following validation rules apply:
+        - Set each byte of |b|'s allocation to zero.
+        - Return |b|.
```

Comment:
This was an unordered list, mostly, up until this return statement. I think it's a clear indication that the algorithm steps must be enumerated with numbers

Reply:
Fair enough. I've updated this PR to move the methods it touches back to numbered bullets and will convert the rest back in a followup. FWIW I left the validation steps and un-numbered since they generally don't have an order to them and it's provides some mildly nice visual differentiation between the validation/algortihm steps.

Reply:
oops just saw this thread. SGTM, and agree the validation rules should be unordered and unnumbered.